### PR TITLE
Added collectionPoint's model, migrations, seeders, schema and resolvers

### PR DESCRIPTION
**Pull request by rickrm**
[Notion task](https://www.notion.so/uwblueprintexecs/9224caeb21ce4350b8a1dc378d393323?v=43f44ac04d734394b7f7edc636c523b4)

Added collectionPoint's table

 **collectionPoint's Columns Added:**
- **id** (Integer, Non-Nullable): numerical identification for the collection point
- **name** (String, Non-Nullable): The name of the collection point
- **createdAt** (Date, Non-Nullable): Gives the day, month, and year that the collection point was created on
- **updatedAt** (Date, Non-Nullable): Gives the day, month, and year that the collection point was updated on
- **eventID** (event, Non-Nullable): every collection point is assigned to an event, eventID stores the appropriate event's id for access to a subfield of that event's name, id, description, and more

**Added:**
- collectionPoint's schema and resolver
- collectionPoint's migration
- collectionPoint's model and associations (belongsTo)
- seeding for collectionPoint

Screen Shot 2020-03-15 at 12 17 22 PM

-Manual testing and checking of new model on Graphiql
## Checklist
- [x] Change title to be more descriptive (shouldn't be just branch name)
- [x] Update `Notion task` link
- [x] `Overview`
- [x] `Changes`
- [ ] \(Optional) Add inline comments in diff
- [x] \(Optional) `Screenshots`
- [x] \(Optional) `Testing` (N/A if nothing changed)
- [ ] \(Optional) `Other Notes`
- [x] Request at least 2 reviewers
- [x] Assign yourself to the PR
- [ ] Make sure build passes

@uwblueprint/paramedics

### CODE REVIEW

File: schema/collectionPoint.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const collectionPointSchema = `
+
+extend type Query {
+
+    collectionPoint(id: Int!): collectionPoint
+    collectionPoints: [collectionPoint]
+
+}
+type collectionPoint {
+
```
Comment (petorsong) [inline schema/collectionPoint.js]: there should be an `id` field in the schema

File: models/collectionPoint.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+module.exports = (sequelize, DataTypes) => {
+  const collectionPoint = sequelize.define('collectionPoint', {
+    name: DataTypes.STRING,
+    //TODO: Add Location (coordinate)
+    createdAt: Date,
+    updatedAt: Date,
+  }, {});
+  collectionPoint.associate = function(models) {
+    // associations can be defined here
+
```
Comment (petorsong) [inline models/collectionPoint.js]: code style: there's a lot of unnecessary empty lines here

File: models/event.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 "use strict";
 const User = require("./user");
+const collectionPoint = require("./collectionPoint");
```
[the remark this answers is not in the document]
Comment (rickrm) [reply models/event.js]: Hi, I left this in when I was testing associations. I'll remove it and push without the dependency.

File: schema/collectionPoint.js (hunk 1 above)
Comment (rickrm) [reply schema/collectionPoint.js]: I forgot to add that, I'll put the id in on my next push.

File: models/collectionPoint.js (hunk 2 above)
Comment (rickrm) [reply models/collectionPoint.js]: Ok, I'll clean up the spacing.

**Comment by JasonYG** [comment]
Adds the `collectionPoints` table to the db and querying functionality, with an association to the events table. I looked over the code, and tested out the changes locally. Everything lgtm!

**Comment by rickrm** [comment]
> see comments for necessary changes
> 
> * also don't check in the `builder` binary file, you can remove it using `git rm`

I removed the binary file

**Comment by rickrm** [comment]
Resolved merge conflicts in graphql.js

**Pull request merged**